base/java-tools/src/com/netscape/cmstools/ca/CACertRequestApproveCLI.java

```diff
@@ -0,0 +1,87 @@
+package com.netscape.cmstools.ca;
+
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.Option;
+import org.dogtagpki.cli.CommandCLI;
+
+import com.netscape.certsrv.ca.CACertClient;
+import com.netscape.certsrv.cert.CertRequestInfo;
+import com.netscape.certsrv.cert.CertReviewResponse;
+import com.netscape.certsrv.request.RequestId;
+import com.netscape.cmstools.cli.MainCLI;
+
+public class CACertRequestApproveCLI extends CommandCLI {
+
+    public static org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(CACertRequestApproveCLI.class);
+
+    CACertRequestCLI certRequestCLI;
+
+    public CACertRequestApproveCLI(CACertRequestCLI certRequestCLI) {
+        super("approve", "Approve certificate request", certRequestCLI);
+        this.certRequestCLI = certRequestCLI;
+    }
+
+    @Override
+    public void printHelp() {
+        formatter.printHelp(getFullName() + " <Request ID> [OPTIONS...]", options);
```

Comment:
As a user, after reading the `--help` section, I assumed that `--input-file` is an optional param while it's not. Should we update this line?

Reply:
Not sure about that. It will make the help line longer. It is a CLI **option**, but an option can be mandatory under certain scenarios. Also, we're considering to make it optional later.